### DISCUSSION TITLE
Cannabis + Lithium

### DISCUSSION THREAD

**Discussion by utaninja** [TripSit/drugs#24] [Drug Interactions]
Surprisingly, I was not able to find much on this combination.

https://www.drugs.com/drug-interactions/cannabis-with-lithium-2758-0-1477-0.html?professional=1 does not seem to indicate anything of concern, and it does not seem I am able to find any study discussing this interaction.
There is a case study of someone possibly getting a negative reaction, but I am not so compelled by this study and it does not seem that there are many reports of this happening any other time to anyone. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7427002/ There is also a ton of factors for this case study and many other medication

It seems that generally any negative reports are associated with people with mental health disorders recognizing that weed is unhealthy for them once they start Lithium and find positive benefits from it.

I don't see any reason to think there is a pharmacological interaction between the two drugs. I am inclined to say low risk + no synergy. I am leaving this open for more comments/input.

**Comment by zelixir25** [comment]
This recently came up with someone indicating cannabis x lithium is dangerous similar to most psychs x cannabis so I started to dig a little further.

The paper cited below contains some anecdotes from people with bipolar who used cannabis to help manage symptoms. Several of them were using it to ease the side effects of lithium. There were no adverse events reported in that paper.

Grinspoon, L., & Bakalar, J. B. (1998). The use of cannabis as a mood stabilizer in bipolar disorder: anecdotal evidence and the need for clinical research. _Journal of psychoactive drugs, 30_(2), 171-177.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/issues/307
